src/Book.java

```diff
@@ -0,0 +1,48 @@
+public class Book {
+//private variables needed to make book objects
+    private String title; 
+    private String author;
+    private String borName;
+    private boolean borrowed;
+    private boolean available;//not really of importance but could be usefull for future extensions
+
+    public Book(String title, String author, String borName, boolean borrowed){//constructer for a book object
+        //sets parameter input to instance variables
+        this.title = title;
+        this.author= author;
+        this.borName = borName;
+        this.borrowed = borrowed;
+        available= !borrowed;//opposite of borrowed
```

Comment:
if available is always the opposite of borrowed, why keep both as instance variables? Just make `getAvailable()` return `!borrowed`